cert_manager/datadog_checks/cert_manager/metrics.py

```diff
@@ -0,0 +1,11 @@
+# (C) Datadog, Inc. 2019
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+
+METRIC_MAP = {
+    'certmanager_certificate_ready_status': 'certificate_ready_status',
+    'certmanager_certificate_expiration_timestamp_seconds': 'certificate_expiration_timestamp',
```

Comment:
nit: same as above ^^
```suggestion
    'certmanager_certificate_expiration_timestamp_seconds': 'certificate.expiration_timestamp',
```